.github/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: "\U0001F41B Bug report"
+about: Create a report to help us improve
+title: ''
+labels: bug report, needs-triage
```

Comment:
I'm guess this is on every repo so I should change it but, it should only say "bug", not "bug report"